### ISSUE TITLE
[Content Author Module 2: Structure] A11y requirements missing

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-curricula#546]
The second learning objective states: "explain accessibility requirements related to content structure"

However, most of the content is presented as if it's a best practice and not a requirement such as, "explain how proper heading hierarchy helps users identify and navigate the different content sections" (emphasis on "helps"). 

I'm not sure what was intended by the learning objective but I'm not seeing that explicit in the content. Are they supposed to be represented by saying, for instance, that authoring tools need to keep the mark-up? Overall, perhaps there's a way to restate the learning objective and/or make some sections (particularly Headings) a bit stronger in their language. As I read through, I wouldn't be able to separate a requirement versus a best practice, for instance.

(As I continue reading, I'm going to say this applies to all the modules that mention "accessibility requirements"; I think this can be more clear throughout or differently stated as an objective.)

**Comment by daniel-montalvo** [comment]
Discussed in[TF meeting 12 July](https://www.w3.org/2022/07/12-wai-curricula-minutes.html#item01).

The TF feels that probably some tweaks to the language are needed to make it more strong. However, the main focus of this resource is not to distinguish between requirements and best practice, but to provide guidance for instructors on how to teach accessibility.

**Comment by daniel-montalvo** [comment]
Hey @MicheleAWilliams-A11y 

As agreed in the TF meeting, I have taken a pass at [Structure](https://content-author-modules--wai-curricula.netlify.app/curricula/content-author-modules/structure/)

I have made the language stronger. I have changed "help" to "enable" throughout.

As said above, we think this addresses the issue, and will mark this as resolved. Feel free to provide specific suggestions if you think this is still not addressed.
